raiden/tests/fixtures/blockchain.py

```diff
@@ -90,9 +90,10 @@ def pregenerate_dag(request, blockchain_type, dagpath):
     )
 
     if blockchain_type == 'geth' and missing_dag:
-        os.makedirs(os.path.dirname(dagpath))
+        dag_folder = os.path.split(dagpath)[0]
```

Comment:
oh, my bad. I missed the `dirname` -- I would still go for a variable, because it will make the call to `geth makedag...` more straight forward.